app/src/main/java/com/spoony/spoony/core/designsystem/component/textField/SpoonyLargeTextField.kt

```diff
@@ -0,0 +1,212 @@
+package com.spoony.spoony.core.designsystem.component.textField
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.focus.focusRequester
+import androidx.compose.ui.focus.onFocusChanged
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.spoony.spoony.R
+import com.spoony.spoony.core.designsystem.theme.SpoonyAndroidTheme
+
+@Composable
+fun SpoonyLargeTextField(
+    value: String,
+    onValueChanged: (String) -> Unit,
+    placeholder: String,
+    modifier: Modifier = Modifier,
+    maxLength: Int = Int.MAX_VALUE,
+    minLength: Int = 0,
+    minErrorText: String? = null,
+    maxErrorText: String? = null
+) {
+    var isFocused by remember { mutableStateOf(false) }
+    var isError: Boolean by remember { mutableStateOf(false) }
+    var selectText: Boolean by remember { mutableStateOf(false) }
```

Comment:
p3) Boolean을 리턴하는 변수의 경우 'is' prefix를 붙이는 네이밍으로 사용해주세요 :)
```suggestion
    var isSelectedText: Boolean by remember { mutableStateOf(false) }
```

Reply:
p1) `selectText`라는 변수명이 모호한 것 같습니다.
아래 사용되는 예시를 보았을 때와 기능 명세를 본다면 maxLength 초과로 입력을 시도하였을 경우 true가 되는 변수로 보입니다.
하지만 selectText라는 변수가 해당 뜻을 나타내는 것으로 해석되지 않네요.

`isOverflowed`처럼 "maxLength를 초과했는가?" 라는 의미가 담긴 변수로 수정하면 좋을 것 같습니다 :)

Reply:
p1) `isError`와 `selectText`의 경우 매번 입력될 때 마다 연산됩니다.
게다가 특정 값이 되기 전까지 true에서 true로 변경되는 일이 잦아요. 이럴때 이전값과 다를 때에만 값을 변경시켜주는 `derivedStateOf`를 적용한다면 불필요한 연산을 줄일 수 있어요. [아티클](https://velog.io/@beokbeok/%EC%96%B8%EC%A0%9C-derivedStateOf%EB%A5%BC-%EC%8D%A8%EC%95%BC%ED%95%A0%EA%B9%8C)을 읽어보시고 적용한다면 도움이 될 것 같습니다 :)

Reply:
변수명 수정했습니다!
derivedStateOF는 앱잼 이후에 시도해보겠습니다ㅠ